src/usr/local/www/services_unbound.php

```diff
@@ -168,6 +178,10 @@
 			$pconfig['outgoing_interface'] = implode(",", $pconfig['outgoing_interface']);
 		}
 
+		if (isset($pconfig['system_domain_local_zone_type']) && !empty($pconfig['system_domain_local_zone_type'])) {
```

Comment:
Considering empty() already check for isset(), isset() is redundant here and can be removed